src/platform/apollolake/include/platform/lib/dma.h

```diff
@@ -47,7 +47,7 @@
 #define DMA_HANDSHAKE_SSP5_TX	12
 #define DMA_HANDSHAKE_SSP5_RX	13
 
-#define dma_chan_irq(dma, chan) (dma_irq(dma) + channel)
+#define dma_chan_irq(dma, chan) (dma_irq(dma) + chan)
 
 int dmac_init(void);
```

Comment:
@akloniex this is an obvious bug. You can even send it as a separate pull request to merge it quicker.

Reply:
Must have worked from the fact that the macro was called always with the name "channel"... Funny how some of these issues just slip like that.

Reply:
@dbaluta @paulstelian97 It's already in separate PR #1751.

Reply:
@tlauda right. Merged!

Reply:
That's why macros suck. I don't understand love for them in c projects. Static inline functions are way better.

Reply:
@jajanusz move fast breaks things! For simple constructs like the one above macros are just OK. That's the beauty of C language. You can never stop having fun with it! :)